pkg/reconciler/workload/synctargetexports/synctargetcompatible_reconcile.go

```diff
@@ -123,6 +123,7 @@ func (e *apiCompatibleReconciler) reconcile(ctx context.Context, syncTarget *wor
 				field.NewPath(gvr.String()), upstreamSchema, downStreamSchema, false)
 			if err != nil {
 				syncTarget.Status.SyncedResources[i].State = workloadv1alpha1.ResourceSchemaIncompatibleState
+				syncTarget.Status.SyncedResources[i].Message = err.Error()
```

Comment:
Do you reset this field to empty when the incompatibility if fixed ?

Reply:
made changes!can you PTAL now?